lib/Alchemy/Phrasea/Model/Manipulator/TokenManipulator.php

```diff
@@ -133,7 +145,10 @@ public function createFeedEntryToken(User $user, FeedEntry $entry)
      */
     public function createDownloadToken(User $user, $data)
     {
-        return $this->create($user, self::TYPE_DOWNLOAD, new \DateTime('+3 hours'), $data);
+        $downloadLinkValidity = $this->getDownloadLinkValidity();
```

Comment:
utiliser ici this->conf->get(keys), avec la valeur par défaut si le setting n'existe pas dans la conf